samples/client/petstore/dart2/petstore_client_lib/lib/api_helper.dart

```diff
@@ -56,6 +56,6 @@ String parameterToString(dynamic value) {
 String _decodeBodyBytes(Response response) {
   final contentType = response.headers['content-type'];
   return contentType != null && contentType.contains('application/json')
-    ? utf8.decode(response.bodyBytes)
```

Comment:
This is getting complex fast

Reply:
Good catch. I'll fix it up tomorrow.

Reply:
Hmm, I checked and it's not what you see above. Now that line shows:

```dart
  ? response.bodyBytes == null ? null : utf8.decode(response.bodyBytes)
```